modules/installing-ocp-agent.adoc

```diff
@@ -10,16 +10,26 @@ The following procedure deploys a single-node {product-title} in a disconnected
 
 .Procedure
 
-. Navigate to the link:https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest[OpenShift mirror site] and download the latest version of the installer and client tarballs that match your operating system.
+. Log in to the {product-title} web console using your login credentials.
 
-. Extract the binary by running the following command:
+. Navigate to link: https://console.redhat.com/openshift/create/datacenter.
 +
-[source,terminal]
-----
-$ tar xvf <file> -C ~/.local/bin
-----
+image::agent_install.png[]
+
+. Click *Run Agent-based Installer locally*. You are directed to the *Install {product-title} on Bare Metal locally with Agent* page.
+
+. Optional: Alternatively, you can also click *Bare Metal (x86_64)* on the *Select an {product-title} cluster type to create* page. You are directed to the *Create an {product-title} Cluster: Bare Metal* page.
+Then, select *Local Agent-based* to go to the *Install {product-title} on Bare Metal locally with Agent* page.
 +
-The file contains the Agent-based Installer.
+image::agent_install_bare_metal.png[]
+
+. Select the operating system and architecture.
+
+. Click the **Download Installer** button to download and extract the install program.
```

Comment:
Not critical, but per ISG, you should remove "button" and just say `Click *Download Installer* to...`